rlberry/manager/utils.py

```diff
@@ -14,3 +16,58 @@ def create_database(db_file):
         connection.close()
         return True
     return False
+
+
+def tensorboard_folder_to_dataframe(path_to_tensorboard_data):
```

Comment:
`tensorboard_to_dataframe` is simpler ? in particular if you implement the change I suggest after

Reply:
done !